app/assets/javascripts/request_dropdown_toggle.js

```diff
@@ -0,0 +1,7 @@
+document.addEventListener("click", (e) => {
```

Comment:
Does this affect the entire webpage? Is there a way we could have it only be on one specific div :o

Reply:
it affects the entire webpage — i tried having it localized to one div but there isn't any that spans the entire page / even if I add one in, there are issues with detecting the click due to different z-indices and whatnot